project/classic/dispatcher/image/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM rockylinux:8
```

Comment:
centos/httpd is 2 years old; outdated; I believe rockylinux is the best choice right now after investigation